coap-gateway/service/signIn.go

```diff
@@ -199,8 +200,8 @@ func subscribeAndValidateDeviceAccess(ctx context.Context, client *Client, owner
 	return client.server.ownerCache.OwnsDevice(ctx, deviceID)
 }
 
-func logSignInError(err error) {
-	log.Errorf("sign in error: %w", err)
+func logSignInError(err error) error {
```

Comment:
`logSignInError` -> `signInError`, it no longer logs.

Reply:
fixed